django_datatables_server_side/views.py

```diff
@@ -49,7 +49,7 @@ def get(self, request, *args, **kwargs):
             return HttpResponseBadRequest()
```

Comment:
I think this modification is not necessary as in the context of `self` there is already `self.request` which could be used.